.github/workflows/fossa.yaml

```diff
@@ -1,18 +1,36 @@
 name: FOSSA
 on:
+  pull_request:
   push:
-    branches:
-      - master
-      - main
 
 jobs:
   fossa:
     runs-on: ubuntu-latest
     if: (github.actor != 'dependabot[bot]' && github.actor != 'snyk-bot')
     steps:
-      - name: Checkout code
-        uses: actions/checkout@v2
+      - uses: actions/checkout@v2
```

Comment:
I don't know why it is failing build. Perhaps can you try v3 for both this and setup-go?